src/resources/views/crud/fields/inc/translatable_icon.blade.php

```diff
@@ -1,3 +1,19 @@
-@if (method_exists($crud->model, 'isTranslatableAttribute') && ($crud->model->isTranslatableAttribute($field['name']) || (isset($field['store_in']) ? $crud->model->isTranslatableAttribute($field['store_in']) : false)) && config('backpack.crud.show_translatable_field_icon'))
-<i class="la la-flag-checkered pull-{{ config('backpack.crud.translatable_field_icon_position') }}" style="margin-top: 3px;" title="This field is translatable."></i>
-@endif
\ No newline at end of file
+@php
+    // if field name is array we check if any of the arrayed fields is translatable
+    $translatable = false;
+    if($crud->model->translationEnabled()) {
+        foreach((array) $field['name'] as $field_name){
+            if($crud->model->isTranslatableAttribute($field_name)) {
+                $translatable = true;
+            }
+        }
+
```

Comment:
```suggestion
        // if the field is a fake one (value is stored in a JSON column instead of a direct db column)
        // and that JSON column is translatable, then the field itself should be translatable
```

Added a suggestion with a comment - if this is what it does.